packages-ci.txt

```diff
@@ -0,0 +1 @@
+git make ccache cpio curl gnupg universal-ctags cscope socat patch gperf quilt byacc python3 python3-pip python3-virtualenv device-tree-compiler tmux lsb-release qemu-system-aarch64 qemu-system-riscv64 libsdl2-2.0-0 libpulse0
```

Comment:
Please remove this file. You can dix missing libraries after installing custom packages with dpkg using `apt-get install -f`.